test/conftest.py

```diff
@@ -23,6 +23,9 @@ def DATA_DIR():
     return Path(__file__).parent / "data"
 
 
+Instrument.file_search_template = str(Path(__file__).parent / "data" / "reflectivity_ui-data" / "REF_M_%s")
```

Comment:
this should be changed not globally but only within the test requiring the change, with a `try..finally` block to make sure the old value of `Instrument.file_search_template` is restored when the test finishes

Reply:
This change is for another test: "test_main_window.py". The instrument path should be changed for all tests.